config/migrations/20230719135250-add-secretaris-mandatees.ttl

```diff
@@ -0,0 +1,62 @@
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix prov: <http://www.w3.org/ns/prov#> .
+
+<http://themis.vlaanderen.be/id/persoon/3571cad5-15c8-4589-a0d8-2de1b6695b51>	a person:Person ;
+	persoon:gebruikteVoornaam	"Jeroen" ;
+	foaf:familyName	"Overmeer" ;
+	mu:uuid	"3571cad5-15c8-4589-a0d8-2de1b6695b51" .
+
+<http://themis.vlaanderen.be/id/persoon/1d4c8917-edc7-495f-a202-55511f459236>	a person:Person ;
+	persoon:gebruikteVoornaam	"Gert" ;
+	foaf:familyName	"Eeraerts" ;
+	mu:uuid	"1d4c8917-edc7-495f-a202-55511f459236" .
+
+<http://themis.vlaanderen.be/id/persoon/1f5486c1-ac11-43e6-a608-7289afad0ad0>	a person:Person ;
+	persoon:gebruikteVoornaam	"Dries" ;
+	foaf:familyName	"Verhaeghe" ;
+	mu:uuid	"1f5486c1-ac11-43e6-a608-7289afad0ad0" .
+
+<http://themis.vlaanderen.be/id/persoon/467ecb38-d8b5-4d41-905b-5c58966dbc86>	a person:Person ;
+	persoon:gebruikteVoornaam	"Raf" ;
+	foaf:familyName	"Suys" ;
+	mu:uuid	"467ecb38-d8b5-4d41-905b-5c58966dbc86" .
+
+<http://themis.vlaanderen.be/id/mandataris/f57a9d61-84ce-438e-b601-a905704ed50d> a mandaat:Mandataris ;
+	mandaat:isBestuurlijkeAliasVan	<http://themis.vlaanderen.be/id/persoon/3571cad5-15c8-4589-a0d8-2de1b6695b51> ;
+	mandaat:start	"2019-10-02T00:00:00Z"^^xsd:dateTime ;
+	org:holds	<http://kanselarij.vo.data.gift/id/mandatarissen/64d19115-015e-42a8-9c73-6650f599ae8c> ;
+	mu:uuid	"f57a9d61-84ce-438e-b601-a905704ed50d" ;
+	dct:title	"Secretaris" .
+
+<http://themis.vlaanderen.be/id/mandataris/8c0f82ac-7424-4fde-be94-d0bdc9e30e1c> a mandaat:Mandataris ;
+	mandaat:isBestuurlijkeAliasVan	<http://themis.vlaanderen.be/id/persoon/1d4c8917-edc7-495f-a202-55511f459236> ;
+	mandaat:start	"2019-10-02T00:00:00Z"^^xsd:dateTime ;
+	org:holds	<http://kanselarij.vo.data.gift/id/mandatarissen/14e0bcfa-b17d-48ea-b48c-effb862a3ae5> ;
+	mu:uuid	"8c0f82ac-7424-4fde-be94-d0bdc9e30e1c" ;
+	dct:title	"Waarnemend Secretaris" .
+
+<http://themis.vlaanderen.be/id/mandataris/66c38df3-8d95-489b-8363-7248dcc17a11> a mandaat:Mandataris ;
+	mandaat:isBestuurlijkeAliasVan	<http://themis.vlaanderen.be/id/persoon/1f5486c1-ac11-43e6-a608-7289afad0ad0> ;
+	mandaat:start	"2019-10-02T00:00:00Z"^^xsd:dateTime ;
+	org:holds	<http://kanselarij.vo.data.gift/id/mandatarissen/14e0bcfa-b17d-48ea-b48c-effb862a3ae5> ;
+	mu:uuid	"66c38df3-8d95-489b-8363-7248dcc17a11" ;
+	dct:title	"Waarnemend Secretaris" .
+
+<http://themis.vlaanderen.be/id/mandataris/c159b4ac-f033-48b6-a881-a13c3391ca6e> a mandaat:Mandataris ;
+	mandaat:isBestuurlijkeAliasVan	<http://themis.vlaanderen.be/id/persoon/467ecb38-d8b5-4d41-905b-5c58966dbc86> ;
+	mandaat:start	"2019-10-02T00:00:00Z"^^xsd:dateTime ;
+	org:holds	<http://kanselarij.vo.data.gift/id/mandatarissen/14e0bcfa-b17d-48ea-b48c-effb862a3ae5> ;
+	mu:uuid	"c159b4ac-f033-48b6-a881-a13c3391ca6e" ;
+	dct:title	"Waarnemend Secretaris" .
+
+<http://themis.vlaanderen.be/id/bestuursorgaan/5fed907ee6670526694a0706> prov:hadMember <http://themis.vlaanderen.be/id/mandataris/f57a9d61-84ce-438e-b601-a905704ed50d> ,
```

Comment:
Zijn de secretarissen deel van de regering? Not sure. @tdn heb jij een mening hierover?

Reply:
Mijn idee hier was dat we net zoals ministers dan kunnen zoeken naar de juiste mandataris van de regeringsamenstelling van een bepaalde periode.
Op zich kunnen we dan bestaande logica opnieuw gebruiken in de frontend en moeten we gewoon andere rollen meegeven om op te filteren.

Reply:
maar mogelijk is dit semantisch niet correct, niet zeker of dat gekoppeld mag worden idd.